bindings/python/native/iota_client/_high_level_api.py

```diff
@@ -0,0 +1,132 @@
+from iota_client.common import send_message_routine, call_client_method
+
+
+class HighLevelAPI():
+
+    @send_message_routine
+    def get_outputs(self, output_ids):
+        """Fetch OutputResponse from provided OutputIds (requests are sent in parallel).
+        """
+        return call_client_method('GetOutputs', {
+            'output_ids': output_ids
+        })
+
+    @send_message_routine
+    def try_get_outputs(self, output_ids):
+        """Try to get OutputResponse from provided OutputIds.
+           Requests are sent in parallel and errors are ignored, can be useful for spent outputs.
+        """
+        return call_client_method('TryGetOutputs', {
+            'output_ids': output_ids
+        })
+
+    @send_message_routine
+    def find_messages(self, message_ids):
+        """Find all messages by provided message IDs.
+        """
+        return call_client_method('FindMessages', {
+            'message_ids': message_ids
+        })
+
+    @send_message_routine
+    def get_balance(self, signer, options=None):
+        """Return the balance for a provided signer and its wallet chain account index.
+           Addresses with balance must be consecutive, so this method will return once
+           it encounters a zero balance address.
+        """
+        return call_client_method('GetBalance', {
+            'signer': signer,
+            'options': options
+        })
+
+    @send_message_routine
+    def get_address_balances(self, addresses):
+        """Return the balance in iota for the given addresses.
+           No seed needed to do this since we are only checking and already know the addresses.
+        """
+        return call_client_method('GetAddressBalances', {
+            'addresses': addresses
+        })
```

Comment:
They got removed today, because with the new unlock conditions it's not clear what the balance is